simapp/app.go

```diff
@@ -198,9 +198,9 @@ func init() {
 }
 
 //go:embed app.yaml
-var appConfigYaml []byte
+var AppConfigYaml []byte
```

Comment:
```suggestion
var appConfigYaml []byte
```

feels more logical to export `AppConfig` and not the yaml